README.md

```diff
@@ -19,6 +19,54 @@ This repository is distributed with [npm][npm]. After [installing npm][install-n
 $ npm install --save primer-primitives
 ```
 
+## Usage
+
+JSON is a highly interoperable format that can be used in many types of projects. You could write scripts to generate CSS, use with plugins for design tools, or import into a theme file for use with CSS-in-JS projects.
+
+### Example
+
+This shows an example for a React.js project. You can import Primer Primitives to provide theme values to a `ThemeProvider`. This is a great way of sharing system styles and can be achieved with popular CSS-in-JS libraries such as [styled-components](https://www.styled-components.com/) and [emotion](https://emotion.sh/).
+
+Here's an example using `styled-components`.
+
+```js
+import React from 'react'
+import styled, { ThemeProvider } from 'styled-components'
+import theme from 'primer-primitives'
+
+const Alert = styled.div`
+  color: ${props => props.theme.colors.green[9]};
+  background-color: ${props => props.theme.colors.green[2]};
+`
+
+const App = props => (
+  <ThemeProvider theme={theme}>
+    <Alert />
+  </ThemeProvider>
+)
+
+```
+
+When used with libraries like [styled-system](https://jxnblk.com/styled-system/), you can make Primer Primitives available to style functions. In this example, we've imported the color function to the component's styles argument. The color values are from the color JSON object in Primer Primitives.
+
+```js
+import React from 'react'
+import styled, { ThemeProvider } from 'styled-components'
+import theme from 'primer-primitives'
+import { color } from 'styled-system'
+
+const Alert = styled.div`
+  ${color}
+`
+
+const App = props => (
+  <ThemeProvider theme={theme}>
+    <Alert color='green.0' bg='green.2' />
```

Comment:
Does styled-system do the magic of converting the string `green.0` to `theme.colors.green[0]`? Or is there any extra set-up you need to do to get that to work besides passing the theme file into `ThemeProvider`?

Reply:
The answer is essentially yes, no extra step. Here is the code if you are curious about the implementation https://github.com/jxnblk/styled-system/blob/master/src/util.js#L22-L23